lib/exports/color-sets.less

```diff
@@ -106,8 +106,8 @@
     200: hsl(23, 87%, 97%);
     300: hsl(27, 90%, 55%);
     400: hsl(27, 90%, 55%);
-    500: hsl(27, 80%, 28%);
-    600: hsl(27, 80%, 28%);
+    500: hsl(27, 87%, 27%);
```

Comment:
@giamir I don't think there's supposed to be a change to the Dark mode colors — only Light and HC light based on my ticket. Dark was ok.

I'm also seeing some different values for orange dark mode everywhere. We did a saturation increase which I'm not seeing in this individual values either: https://stackexchange.slack.com/archives/C05RZ6Y8NKD/p1698164022556159?thread_ts=1698162819.819939&cid=C05RZ6Y8NKD

Reply:
It's possible I grabbed the color-sets.less file from the wrong place. I would pull fresh from what's in stacks today and change the orange values only.

Reply:
Sometimes Github is collapsing files and causes confusion.
<img width="1299" alt="Screenshot 2023-11-03 at 09 02 22" src="https://github.com/StackExchange/Stacks/assets/6231616/a6e22638-99bf-479c-a9b9-6625de6d3ddf">

These are the only changes I made yesterday in terms of source code files.
<img width="1096" alt="Screenshot 2023-11-03 at 08 57 50" src="https://github.com/StackExchange/Stacks/assets/6231616/66821a17-c69a-4006-810f-663735b66448">
As you can see only `orange` `orange-hc` has changed as mentioned in the ticket.
This branch is based off develop and it contains all the latest changes already.